Sources/SwiftHtml/Attributes/Global.swift

```diff
@@ -39,20 +39,23 @@ private extension String {
     }
 }
 
-private extension Array where Element == String {
+private extension Set where Element == String {
     
-    var classString: String {
-        filter { !$0.isEmpty }.joined(separator: " ")
-    }
+    var classString: String { joined(separator: " ") }
 }
 
 public extension Tag {
         
     // MARK: - class management
 
     /// find an existing class attribute and return the value as an array of strings or an empty array
-    private var classArray: [String] {
-        node.attributes.first { $0.key == "class" }?.value?.classArray ?? []
+    var classes: Set<String> {
```

Comment:
Transform class attribute value into `Set<String>.` Also, make `var` public to allow testing and possibly valuable to library user